contents/articles/2017-01_retina-mapping/index.md

```diff
@@ -0,0 +1,24 @@
+--- 
+title: Retinotopic to Spatiotopic Mapping in Blind Patients Implanted with Retinal Prosthesis
+date: Mon Jan 30 2017 10:36:53 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Blind patients implanted with a retinal prosthesis fitted to a modified Pupil headset to measure the accuracy in mapping stimulated retinal signals to the world coordinates..."
+featured_img: "../../../../media/images/blog/epiretinal-array-dark.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/epiretinal-array.png" 
+---
+
+<div align="center">
+	<img src="../../../../media/images/blog/thumb/epiretinal-array.png" class='Feature-image u-padBottom--1' alt="Epiretinal Array of Electrodes" style="width: 50%;">
+	<div class="small u-padBottom--3">Image Source: [Retinotopic to Spatiotopic mapping in blind patients implated with the Argus II retinal prosthesis](http://iovs.arvojournals.org/article.aspx?articleid=2597840#149774812)
+	</div>
+</div>
+
+The observation that electrical stimulation of retina in blind patients can create visual percept led to the development of retinal prosthesis devices implanted in blind patients. For the percept to be useful, the electrical stimulation should convey information to the brain with the correct spatial coordinates.
```

Comment:
Current retinal implant systems (like the Argus II) use images from a scene in order to create electrical stimulations on a patient's retina. These systems have been shown to be helpful for blind people in accomplishing activities of daily living. However, current systems have a narrow field of view (18 degrees x 11 degrees) which require the participant to move their head in order to scan a scene. Integrating eye tracking will enable patients with retinal implants to use more natural eye movements to scan the scene.